src/actions/blocks.js

```diff
@@ -84,7 +84,7 @@ export const forgingTimesRetrieved = nextForgers => async (dispatch, getState) =
   latestBlocks.forEach((item, index) => {
     if (index > forgedInRoundNum && !forgingTimes[item.generatorPublicKey]) {
       // if I can't find it in the previous round
-      if (index >= forgedInRoundNum + 101 && latestBlocks.indexOf(item) === index) {
+      if (index >= forgedInRoundNum + 103 && latestBlocks.indexOf(item) === index) {
```

Comment:
I am looking at this statement and I don't understand why is it there. `latestBlocks.indexOf(item) === index` This should always be true or am I missing something?